### PR TITLE
data: HameedKareem

### PR DESCRIPTION
**Pull request by maxton4life**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
[ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: HameedKareem.json
Comment (rizahmeds) [inline HameedKareem.json:44]: Please remove this extra closing curly brace.

**Review by sital002 — COMMENTED** [review]
Please put the file inside of data folder, instead of ```HameedKareem.json ``` it should be ```data/maxton4life.json```. Your filename must match your github username here is the quick guide link https://www.biodrop.io/docs/quickstart-json.

**Comment by maxton4life** [comment]
Here are the problems I face while working on Biodrop : 
1. I thought I was working on the forked repo before committing the JSON file.
2. I also think I was working on the data directory, but to my surprise, the code is not there. 
3. I have since replicated this on the forked repo and also made a pull request. I haven't been replied or notified.

**Comment by eddiejaoude** [comment]
> 1. I thought I was working on the forked repo before committing the JSON file.
> 2. I also think I was working on the data directory, but to my surprise, the code is not there.

We have good documentation with steps to follow to do this, plus a video there also if you prefer. Here is the link https://www.biodrop.io/docs/quickstart-json

> 3. I have since replicated this on the forked repo and also made a pull request. I haven't been replied or notified.

Not sure what you mean @maxton4life , the community are trying to help on your pull request here, I can see the conversation above

File: HameedKareem.json
Comment (maxton4life) [reply HameedKareem.json:44]: How do I access the file cant seem to edit it?

Comment (sital002) [reply HameedKareem.json:44]: image
Click on edit file and then you can edit the file.

**Review by rizahmeds — APPROVED** [review]
Looks Good!

**Comment by SaraJaoude** [comment]
> Looks Good!

@rizahmeds please do not approve a PR without checking it first as this causes extra notifications for maintainers and confusion to the author.  As mentioned above this one needs to be moved to the data folder and the file name needs changing - it is not ready to be merged.

**Comment by maxton4life** [comment]
Please how do I move this file to the data folder? Do I need to clone my branch locally? Therefore moving the file into the folder 📂.

**Comment by sital002** [comment]
> Please how do I move this file to the data folder? Do I need to clone my branch locally? Therefore moving the file into the folder 📂.
>
@maxton4life  Here are steps that you can follow to move your file inside of data folder.

Step 1 : Go to the file changed section from this PR  and click on Edit File. This will redirect you to your forked repo.
image

 Now you are in your forked repo go the input field Hammed.json
go to the begginnig and type ```data/```  this will put you file into the data folder and don't forget to changed the username to your github username. 
image

image

At last after all the changes have been made you can commit don't create a new Pull Request instaed directly commit to this branch.

File: HameedKareem.json
Comment (maxton4life) [reply HameedKareem.json]: I can't find seem to find the file on my forked. It's not in the main branch.

**Comment by sital002** [comment]
Please change the filename to your github username instead of ```data/Hameed.json ``` it should be ```data/maxton4life.json``` to get your profile merged.
We have mentioned this in our quick start guide https://www.biodrop.io/docs/quickstart-json (in section 7 ).

**Comment by maxton4life** [comment]
Okay.
I am on it.

**Review by SaraJaoude — APPROVED** [review]
looks good - I will merge

**Pull request merged**
